news.xml

```diff
@@ -1,6 +1,117 @@
 <?xml version="1.0" encoding="utf-8" ?>
 
 <news>
+  <item>
+    <pubDate>TODO</pubDate>
```

Comment:
I guess this should have been replaced by the actual current timestamp right before merging?